setup.py

```diff
@@ -24,14 +24,15 @@
     use_scm_version={
         'version_scheme': 'guess-next-dev',
         'local_scheme': 'dirty-tag',
-        'write_to': 'zarr/version.py'
+        'write_to': 'zarr/version.py',
     },
     setup_requires=[
         'setuptools>=38.6.0',
-        'setuptools-scm>1.5.4'
+        'setuptools-scm>1.5.4',
     ],
     extras_require={
         'jupyter': [
+            'jupyter',
```

Comment:
Should we just do `notebook`? This seems to install `qtconsole` and other things users may not need/want.

ref: https://pypi.org/project/jupyter/#description

Reply:
Good point, I didn't realise `pip install notebook` was a thing. Changing it now. :)